src/ivcam/sr300.cpp

```diff
@@ -69,34 +73,36 @@ namespace librealsense
         std::vector<platform::uvc_device_info> chosen;
         std::vector<std::shared_ptr<device_info>> results;
 
-        auto correct_pid = filter_by_product(uvc, { SR300_PID, SR300v2_PID });
+        auto correct_pid = filter_by_product(uvc, { SR300_PID, SR300v2_PID, SR306_PID });
         auto group_devices = group_devices_by_unique_id(correct_pid);
         for (auto& group : group_devices)
         {
-            if (group.size() == 2 &&
-                mi_present(group, 0) &&
-                mi_present(group, 2))
-            {
-                auto color = get_mi(group, 0);
-                auto depth = get_mi(group, 2);
-                platform::usb_device_info hwm;
+            platform::usb_device_info hwm;
+            platform::uvc_device_info dev0;
+            platform::uvc_device_info dev2;
 
-                if (ivcam::try_fetch_usb_device(usb, color, hwm))
-                {
-                    auto info = std::make_shared<sr300_info>(ctx, color, depth, hwm);
-                    chosen.push_back(color);
-                    chosen.push_back(depth);
-                    results.push_back(info);
-                }
-                else
+            if (mi_present(group, 0))
```

Comment:
why the `if (group.size() == 2` is removed?

Reply:
It is not relevant after refining the code.
I added the condition if (mi_present(group, 2)) below which makes the condition if (group.size() == 2) redundant